docs/reference-guides/theme-json-reference/theme-json-living.md

```diff
@@ -204,19 +204,6 @@ Generate custom CSS custom properties of the form `--wp--custom--{key}--{nested-
 ## Styles
 
 
-### background
-
-Background styles
-
-| Property  | Type   |  Props  |
-| ---       | ---    |---   |
-| backgroundImage | string, object |  |
-| backgroundPosition | string, object |  |
-| backgroundRepeat | string, object |  |
-| backgroundSize | string, object |  |
-
----
-
```

Comment:
Unfortunately, if we update the JSON schema correctly, the background section will be missing from the documentation. This is because the script that generates the documentation [relies only on `stylesProperties`](https://github.com/WordPress/gutenberg/blob/1cfc442d7da593b98408e6776770fd91008e77bf/bin/api-docs/gen-theme-reference.js#L245) and does not consider which level the properties are available.

We may need to make major changes to the logic of the script, or the overall design of the JSON schema, to ensure that it also generates the documentation correctly.

Personally, I think it's better not to have a background property at this point, because if there is a background property in the document, there is a risk of misunderstanding that this property is available at all levels.

Reply:
Agree. Thanks for noticing this detail!